topics/materials-science/tutorials/muon-stopping-sites-muairss-uep/workflows/main_workflow.ga

```diff
@@ -0,0 +1,320 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "Simple workflow containing PyMuonSuite tools for training purposes.",
+    "creator": [
+        {
+            "class": "Organization",
+            "name": "Muon Spectroscopy Computational Project",
+            "url": "https://muon-spectroscopy-computational-project.github.io"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "MIT",
+    "name": "Workflow to estimate muon stopping site using UEP (imported from uploaded file)",
+    "steps": {
+        "0": {
+            "annotation": "Parameters for PyMuonSuite. This can be generated from the dedicated PyMuonSuite AIRSS Configure tool.",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Parameters for PyMuonSuite. This can be generated from the dedicated PyMuonSuite AIRSS Configure tool.",
+                    "name": "pymuon-suite parameters file (*.yaml)"
+                }
+            ],
+            "label": "pymuon-suite parameters file (*.yaml)",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 0,
+                "top": 191.0
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"format\": [\"yaml\"], \"tag\": \"\"}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "b50f9547-8cac-4dda-83b7-1d9e49f82c8a",
+            "workflow_outputs": []
+        },
+        "1": {
+            "annotation": "File containing the electronic density of the structure of interest.",
+            "content_id": null,
+            "errors": null,
+            "id": 1,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "File containing the electronic density of the structure of interest.",
+                    "name": "CASTEP formatted electronic density "
+                }
+            ],
+            "label": "CASTEP formatted electronic density ",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 278,
+                "top": 0.0
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"format\": [\"castep\"], \"tag\": \"\"}",
```

Comment:
Steps 1 and 2 are the wrong way round on formats - not sure how this happened. Hard-coding never feels like a great idea for these... perhaps re-download the latest version of the public workflow and use that (not sure when this file was created)
```suggestion
            "tool_state": "{\"optional\": false, \"format\": [\"den_fmt\"], \"tag\": \"\"}",
```